src/lib.rs

```diff
@@ -13,7 +13,7 @@
 //!
 //! const PATH: &str = "/tmp/some_file/s8329894";
 //! # fn main() -> Result<(), io::Error> {
-//! let lockfile = Lockfile::create(PATH)?;
+//! let lockfile = Lockfile::create(PATH).unwrap();
```

Comment:
Why unwrap here?

Reply:
Because `Lockfile::create` and the drop have different error types. I unwrap to avoid having to do fancy stuff with errors. It's only an example, in practice you'd handle the error.

Reply:
I'll merge since this is just a doc example and can be changed later without breaking compatibility.